backend/src/bundles/users/user.service.ts

```diff
@@ -24,6 +24,10 @@ class UserService implements Service {
         return await this.userRepository.findByEmail(email);
     }
 
+    public async findById(userId: number): Promise<UserEntity | null> {
+        return await this.userRepository.findById(userId);
+    }
+
```

Comment:
Modify method find instead of adding this

Reply:
Only in user service or in user repository as well?

Reply:
Modified only in user service for now

Reply:
please also modify it in the repository

Reply:
done